### DISCUSSION TITLE
Possibilité d'action massive pour modifier ou corriger les horaires, nombres de place, information.

### DISCUSSION THREAD

**Discussion by MagaliMoserNoriaIRL** [unistra/esup-immersup-demandes#9] [4- Saisie de l'offre d'immersion]
Tout est dans le titre... C'est la demande d'un référent lycée, il souhaiterait pouvoir impacter plusieurs créneaux d'immersion pour les modifier en une seule fois (par exemple, s'il ne souhaite plus accueillir dans la même salle, etc).

**Comment by efinkbeiner5** [comment]
Solution envisagée : pouvoir sélectionner plusieurs objets, afficher le formulaire de modification avec en italique les données non identiques et non italique les données communes. A l'enregistrement, On les prévient qu'on modifie TOUS les objets et l'appli enregistre tous les objets en une fois avec les nouvelles données saisies. 
Attention cela va ajouter une colonne au début de la table des créneaux, attention aux nombres de mails générés si on coche la case de notifier les inscrits. Est-ce qu'on limite le nombre de créneaux sélectionnés ? 
Attention, cela doit être bien utilisé. 
Chiffrage : Modéré

**Reply by efinkbeiner5** [reply]
NE pas pouvoir modifier les dates en action massive

**Comment by efinkbeiner5** [comment]
Fait dans la version 4 en 2024
